src/OSPSuite.Infrastructure.Import/Core/DataSource.cs

```diff
@@ -213,19 +213,20 @@ private void validateUnitsSupportedAndSameDimension(IReadOnlyList<ColumnInfo> co
                   //if unit comes from a column
                   if (column.Key.Column.Dimension == null)
                   {
-                     var dimensionOfFirstUnit = columnInfo.SupportedDimensions.FirstOrDefault(x => x.HasUnit(column.Value.ElementAt(0).Unit));
+                     var firstNonEmptyUnit = column.Value.FirstOrDefault(x => !string.IsNullOrEmpty(x.Unit));
+                     var dimensionOfFirstUnit = columnInfo.SupportedDimensions.FirstOrDefault(x => x.FindUnit(firstNonEmptyUnit.Unit, true) != null);
```

Comment:
can you add , ignoreCase: true instead of true? It makes the code much easier to understand
see https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/blob/develop/src/OSPSuite.Core/Domain/UnitSystem/DimensionFactory.cs#L125 in code base